### DISCUSSION TITLE
Frigate in Docker on Debian - Newbie

### DISCUSSION THREAD

**Discussion by gardenvariety6** [blakeblackshear/frigate#6009] [Ask A Question] [answered]
Hello,

I had Frigate running successfully in my Home Assistant running on RPi, now I am trying to give Frigate its own dedicated hardware with a Coral TPU.  I have Portainer setup with Docker which can successfully pull the image from the internet. Unfortunately, my docker container cannot connect to anything on the local network. This means it cannot connect to the cameras. I have done much research, including both Docker and Portainer sites but I still can't figure out how to make it connect to the local network. All the documentation and videos setting up Frigate in this way don't seem to cover this step.

Can anyone point me in the right direction? I suspect I need to create a Network, perhaps macvlan, but my attempts so far havent worked. Maybe there are some sample configurations or documentation someone can point me to?

Thanks very much.

Roger

**Comment by kdill00** [comment]
It’s much easier to get it started via command line or docker compose first. Then manage it with portainer if it’s ever needed. Have you attempted these steps from the documentation?

https://docs.frigate.video/frigate/installation


If you are still stuck, please give a run down of where you are at. If this is a raspi with home assistant frigate and a usb coral? Ect.

**Reply by gardenvariety6** [reply, marked as answer]
I have followed those documentation... I had changed a few things in an attempt to get things working but was never able to get the local network access. 

I decided to do a fresh install, but this time, docker-desktop and amazingly it all works!

Thank you for taking the time to respond.
